src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -3607,6 +3603,11 @@ public enum RisingRivals implements LogicCardInfo {
               checkFaint()
             }
             shuffleDeck()
+            delayed {
```

Comment:
Regarding the above TODO. Maybe we could (if the Rotom is active)
1. Make the bench 1 bigger
2. Create a Fake PCS on the bench
3. Switch with the Fake PCS
4. Switch back to Active
5. Remove the Fake PCS
6. Reduce the bench size back

Most effects unregister after FALL_BACK after all.

Reply:
This could work except there are other effects that trigger on retreat/switch. This is only my personal preference, but if the card will have bugs either way, I would prefer if they stemmed from an unimplemented feature rather than a janky implementation.

Reply:
Yeah, I agree. It was just an idea I had thought of, but you make a good point.